crates/external_attestation/coordinator/src/chain_info.rs

```diff
@@ -1,11 +1,11 @@
 use hdk::prelude::*;
 
-#[hdk_extern]
+//#[hdk_extern]
```

Comment:
Why are these disabled?

Reply:
again these common holochain functions didnt merit a separate zome crate.. but because they use the HDK i could not add them into shared_utils without dependency problems..   We could create a new zome crate to collate all these small functional units..  personally without any validation,  I dont really see the point in doing that.